update version vendure and fixed bug mac m1

lucasvieceli opened the pull request:
update version vendure and fixed bug mac m1

Commit:
update version vendoure and fixed bug mac m1

Commit:
add file setup.ts

Commit:
add package

Commit:
add package

michaelbromley commented:
Hi @lucasvieceli 
Thank you for your contribution!
I have a few points / questions:

1. Would you be able to do this PR _without_ formatting all the files? It makes it really hard to see the important changes. 
2. You have removed a number of devDependencies which are required during setup like `fs-extra` and `@vendure/create`. Why?
3. Why is the `RUN yarn build` command commented out in the vendure-docker file?

Commit:
- added @vendure/create in package.json
- modified setup.ts, because when starting the "work" service it reported that it couldn't find the assets folder, in the "server" service it will only create the assets folder templates email. And in the "server" you will populate the tables and the images of the products
- Added fs-extra

lucasvieceli commented:
In this pull request, I created a project on my machine with the new version, and passed it to the docker repository, the folder "src", the file "tsconfig.json" and "migrations.ts"

I made a new commit with some changes

- Added @vendure/create in package.json
- Modified setup.ts, because when starting the "work" service it reported that it couldn't find the assets folder, in the "server" service it will only create the assets folder templates email. And in the "server" you will populate the tables and the images of the products
- Added fs-extra
- Removed "yarn build", because it is running ts-node directly
- I will remove spaces as requested


sorry english (google)

Commit:
add .editorconfig

Commit:
fixed .editorconfig single quote

Commit:
fixed urls email
